# Message describing a detection of a station

# Unique string of the detection
string 			 name

# Index of the center of the detection
geometry_msgs/Point   center

# Angle to extend the costs
float64 angle

# Magnitude to extend the costs
float64 magnitude
